src/devices/cpu/upd7810/upd7810.cpp

```diff
@@ -1476,10 +1476,11 @@ void upd7810_device::handle_timers(int cycles)
 			m_adcnt -= m_adtot;
 			switch (m_adout)
 			{
-				case 0: CR0 = m_tmpcr; break;
-				case 1: CR1 = m_tmpcr; break;
-				case 2: CR2 = m_tmpcr; break;
-				case 3: CR3 = m_tmpcr; break;
+				// volfied code checks bit 0x80, old code set bit 0x01, TODO: verify which bits are set on real hw
```

Comment:
There are diagrams in the uPD7810 manual which shows that the CRx registers are 8-bit wide.
Maybe m_tmpcr is supposed to contain an aggregated result at this point?

Reply:
it seems more likely that it's like this to me based on what I can see, but yeah, it seems a bit redundant for all bits to be the same, hence the comment so that if it is revisited later the reason for it being this way is known